libs/ledger-live-common/src/hw/actions/app.ts

```diff
@@ -374,22 +355,20 @@ const reducer = (state: State, e: Event): State => {
 
     case "opened":
       return {
-        requestQuitApp: false,
-        requestOpenApp: null,
-        requiresAppInstallation: null,
-        allowOpeningGranted: false,
-        allowOpeningRequestedWording: null,
-        allowManagerGranted: false,
-        allowManagerRequested: false,
-        device: state.device,
-        error: null,
+        ...getInitialState(state.device, state.request),
         isLoading: false,
-        unresponsive: false,
-        isLocked: false,
+        device: state.device,
         opened: true,
         appAndVersion: e.app,
         derivation: e.derivation,
 
+        deviceInfo: undefined,
+        latestFirmware: undefined,
+        installingApp: undefined,
+        listingApps: undefined,
+        installQueue: undefined,
+        itemProgress: undefined,
```

Comment:
all those values are already defined in getInitialState, but why setting them `undefined` instead of their initial values boolean or null?

Reply:
It's to keep the same behavior as before, these values were not initialized using the default state.